packages/core/usePictureInPicture/index.md

```diff
@@ -0,0 +1,32 @@
+---
+category: Browser
+---
+
+# usePictureInPicture
+
+Reactive handler for the [Picture-in-Picture API](https://developer.mozilla.org/en-US/docs/Web/API/Picture-in-Picture_API).
```

Comment:
To ensure consistency of Documentation (as with other functions):

Reactive  [Picture-in-Picture API](https://developer.mozilla.org/en-US/docs/Web/API/Picture-in-Picture_API).

Reply:
Done ✅